articles/sustainable-ai-with-tinyml/index.md

```diff
@@ -0,0 +1,155 @@
+
+But why do we need to pay more attention to TinyML?
+
+### Motivation for Tiny Machine Learning
+
+The biggest reason why TinyML deserves more attention is sustainability. Environmental sustainability refers to the responsible interaction with the planet to maintain natural resources. The goal of said sustainability is to ensure that future generations have the natural resources required to live not only an equal but better life than current generations.
+
+Sustainability is being threatened by some technology trends. Machine learning models were small enough to be run on basic local machines. Data produced from our daily digital activity keeps growing at an incredible rate. Machine learning and big data are based on extracting valuable information from very large amounts of data.
+
+The evolution of Graphics Processing Units (GPUs) has been making it possible to deal with much larger datasets. Cloud-based services (like Software-as-a-Service platforms) also emerged. However, the convergence of the increase in digital data produced with the advancement of GPUs and the emergence of cloud services meant that the size of algorithms could grow. And they did grow exponentially. As did their computational requirements. For example, consider state-of-the-art language models like [BERT](https://arxiv.org/abs/1810.04805), [GPT-2]( https://cdn.openai.com/better-language-models/language_models_are_unsupervised_multitask_learners.pdf) and [GPT-3](https://www.section.io/engineering-education/introducing-gpt3/#:~:text=Introduction%20to%20GPT3%20October%2010%2C%202020%20Generative%20pre-trained,people%20with%20little%20tech%20background%20to%20develop%20applications.). Bert Large was released in 2018 with 355 million parameters to its name. GPT-2 followed in 2019 with 1.5 billion parameters. This year, GPT-3 blew them both out of the water with 175 billion parameters. This shows that the size of these language models in particular is increasing by at least a factor of 10 annually. In fact, model sizes are already outpacing GPU memory. To read more on these models check out this [article](https://www.section.io/engineering-education/introducing-gpt3/#:~:text=Introduction%20to%20GPT3%20October%2010%2C%202020%20Generative%20pre-trained,people%20with%20little%20tech%20background%20to%20develop%20applications.).
+
+Having such large models means that one would need great amounts of power to run and maintain the models. Power usage has been increasing with the scary increase in the size of models. It is estimated that GPT-3 used about 3 GWh of electricity to train. This is comparable to the output of about [three nuclear plants](https://www.britannica.com/technology/nuclear-power). You can definitely imagine the size of carbon emission as a result of such power usage. The carbon footprint as a result is very large and will keep increasing if these trends continue.
```

Comment:
Output of nuclear plant for how long? If that Info is unavailable, it would be better to restructure the sentence.